try:
    A=int(input())
except:
    print('error occured')
finally:
    print('exception handling completed')
